templates/taxi/car_detail.html

```diff
@@ -1,12 +1,16 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }} {{ car.model }})</p>
```

Comment:
The parentheses around the car details in the paragraph might be unnecessary and could be removed for cleaner output unless they serve a specific purpose.